### ISSUE TITLE
Image gallery close button doesn't get initial focus

### ISSUE THREAD

**Issue by davidpmccormick** [wellcomecollection/wellcomecollection.org#11580]

**Comment by davidpmccormick** [comment]
When you open an image gallery (e.g. [Exhibition highlights on this page](https://wellcomecollection.org/exhibitions/hard-graft-work-health-and-rights)) the close button should be the thing that has focus so that a keyboard user can easily close the gallery again. Currently they have to shift-tab back up through the whole gallery before reaching the button
